src/directives/sortable.js

```diff
@@ -96,9 +102,13 @@ ngeo.sortableDirective = function($timeout) {
               dragListGroup.dispose();
             }
 
+            if (options['dragHoverClass'] !== undefined) {
+              dragHoverClass = options['dragHoverClass'];
+            }
+
             dragListGroup = new goog.fx.DragListGroup();
             dragListGroup.addDragList(element[0],
-                goog.fx.DragListDirection.DOWN);
+                goog.fx.DragListDirection.DOWN, true, dragHoverClass);
```

Comment:
Any example of a usage of the `dragHoverClass` options? Looks like it's not used at all.
I'd rather add it if we really need it.